source/codegen/templates/service_helpers.mako

```diff
@@ -19,7 +20,11 @@ ${initialize_input_params(function_name, parameters)}
       };
       uint32_t session_id = 0;
       const std::string& session_name = request->session_name();
+      %if 'custom_close' in function_data:
+      auto cleanup_lambda = [&] (uint32_t id) { library_->${function_data['custom_close']}; };
+      %else:
       auto cleanup_lambda = [&] (uint32_t id) { library_->${config['close_function']}(id); };
+      %endif
```

Comment:
Consider factoring out the close call into a python variable in the `<% %>` block above:
```python
close_function_call = function_data['custom_close'] if 'custom_close' in function_data else f"{config['close_function']}(id)"
```
```cpp
auto cleanup_lambda = [&] (uint32_t id) { library_->${close_function_call}; };
```
```

Reply:
Yes, changed it.